addons/mail/static/src/models/autocomplete_input_view/autocomplete_input_view.js

```diff
@@ -0,0 +1,232 @@
+                    return 'o_DiscussSidebarCategory_newChannelAutocompleteSuggestions';
+                }
+            }
+            if (this.messagingMenuAsMobileNewMessageInput) {
+                return 'o_MessagingMenu_mobileNewMessageInputAutocomplete';
+            }
+        },
+        /**
+         * @private
+         * @returns {boolean|FieldCommand}
+         */
+        _computeIsHtml() {
+            if (this.discussSidebarCategoryAsAddingItemInputViewOwner) {
+                return this.discussSidebarCategoryAsAddingItemInputViewOwner === this.messaging.discuss.categoryChannel;
+            }
+            return clear();
+        },
+        /**
+         * @private
+         * @returns {string|FieldCommand}
+         */
+        _computePlaceholder() {
+            if (this.chatWindowOwnerAsNewMessageFormInputOwner) {
+                return this.env._t("Search user...");
+            }
+            if (this.discussAsChannelMobileAddItemHeaderOwner) {
+                return this.env._t("Create or search channel...");
+            }
+            if (this.discussAsChatMobileAddItemHeaderOwner) {
+                return this.env._t("Search user...");
+            }
+            if (this.discussSidebarCategoryAsAddingItemInputViewOwner) {
+                return this.discussSidebarCategoryAsAddingItemInputViewOwner.newItemPlaceholderText;
+            }
+            if (this.messagingMenuAsMobileNewMessageInput) {
+                return this.env._t("Search user...");
+            }
+            return clear();
+        },
+    },
+    fields: {
+        chatWindowOwnerAsNewMessageFormInputOwner: one('ChatWindow', {
+            inverse: 'newMessageFormInputView',
+            readonly: true,
+        }),
+        /**
+         * States the OWL component of this autocomplete input view.
+         */
+        component: attr(),
+        customClass: attr({
+            compute: '_computeCustomClass',
+            default: '',
+        }),
+        discussAsChannelMobileAddItemHeaderOwner: one('Discuss', {
+            inverse: 'channelMobileAddItemHeaderInputView',
+            readonly: true,
+        }),
+        discussAsChatMobileAddItemHeaderOwner: one('Discuss', {
+            inverse: 'chatMobileAddItemHeaderInputView',
+            readonly: true,
+        }),
+        discussSidebarCategoryAsAddingItemInputViewOwner: one('DiscussSidebarCategory', {
+            inverse: 'addingItemInputView',
+            readonly: true,
+        }),
+        doFocus: attr({
+            default: false,
+        }),
+        isFocused: attr({
+            default: false,
+        }),
+        isHtml: attr({
+            compute: '_computeIsHtml',
+            default: false,
+        }),
+        messagingMenuAsMobileNewMessageInput: one('MessagingMenu', {
+            inverse: 'mobileNewMessageInputView',
+            readonly: true,
+        }),
```

Comment:
This case is lacking an owner/identifying field/compute.

Reply:
Done (also fixed all other "owner" field names)

Reply:
(and it's an owner, so no compute and properly readonly)